majordodo-core/src/test/java/majordodo/replication/ReplicatedCommitLogSimpleTest.java

```diff
@@ -70,6 +75,13 @@ public void test() throws Exception {
                 LogSequenceNumber logStatusEdit2 = log.logStatusEdit(edit2);
                 LogSequenceNumber logStatusEdit3 = log.logStatusEdit(edit3);
                 LogSequenceNumber logStatusEdit4 = log.logStatusEdit(edit4);
+                
+                zkServer.bookie.getBookie().getLedgerManagerFactory().newLedgerManager().readLedgerMetadata(log.getCurrentLedgerId(), (i, t) -> {
```

Comment:
Not sure this is the best way. you are accessing the internals of the Bookie, so you are not using a public API

You should create a BookKeeper client and than use BookKeeper.openLedgerNoRecovery(...) and then read Metadata

ReplicatedCommitLog should have a BookKeeper client, better to use that one